### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by RileyManda**
I have completed work on the Portfolio’s contact section in accordance with the project requirements.

Here are the main changes I made:

- Collaborated with a Pair-Coding partner who created the form to match the CSS styling I provided.
-Collaborated with Pair-Coding to ensure the following form conditions were met:
**-Utilized form spree to ensure data from the form is being collected by [formspree](https://formspree.io/).**
**- Name field accepts up to 30 chars**
**- Name field accepts up to 30 chars.**
**- Email field accepts only emails.**
**- Message field accepts up to 500 chars.**
**- All fields have placeholder text.**
**- All fields have the correct type.**
**- A user cannot submit the form if one or more of the fields is empty.**
**-Responsiveness:**:Made sure the components look great and function well on different devices with media queries.**

Please review the changes and let me know if there are any issues or suggestions for improvement.

I look forward to your feedback.

Thank you!
Best regards,

### CODE REVIEW

File: style.css
Comment (AbelG101) [inline style.css:1]: https://github.com/RileyManda/portfolio-website/blob/11bead5cfeaf890c21612829cc53956bebdb20b3/style.css#L640-L642
- [x] Ouch! this is really close but it seems like you forgot to implement the `:hover` & `:active` states of the "Get in touch" button. So please make sure to add that
    - Here's the Figma design as a reminder:
      firefox_kz6PEcoelc

File: .vscode/settings.json
Comment (AbelG101) [inline .vscode/settings.json:1]: **THIS COMMENT IS REGARDING YOUR README AND NOT THIS FILE**
- [x] Kindly remove any section descriptions from your README as they're only there to guide the author on what to place on each section and should be removed from the final draft. ✔
  https://github.com/RileyManda/portfolio-website/blob/11bead5cfeaf890c21612829cc53956bebdb20b3/README.md?plain=1#L147
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @RileyManda 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  UI is looking on point ✔
- Met 99% of the requirements ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RileyManda** [comment]
> I have completed work on the Portfolio’s contact section in accordance with the project requirements.
> 
> Here are the main changes I made:
> 
> * Collaborated with a Pair-Coding partner who created the form to match the CSS styling I provided.
>   -Collaborated with Pair-Coding to ensure the following form conditions were met:
>   **-Utilized form spree to ensure data from the form is being collected by [formspree](https://formspree.io/).**
>   **- Name field accepts up to 30 chars**
>   **- Name field accepts up to 30 chars.**
>   **- Email field accepts only emails.**
>   **- Message field accepts up to 500 chars.**
>   **- All fields have placeholder text.**
>   **- All fields have the correct type.**
>   **- A user cannot submit the form if one or more of the fields is empty.**
>   **-Responsiveness:**:Made sure the components look great and function well on different devices with media queries.**
> 
> Please review the changes and let me know if there are any issues or suggestions for improvement.
> 
> I look forward to your feedback.
> 
> Thank you! Best regards,

I have completed itirations on the Portfolio’s contact section as requested by the reviewer.

Here are the itirations I made:

**-Removed description/instruction text readme as recommended by the review.**
**-Added hover and active states to the contactform button and other buttons in the project.**

Please review the changes and let me know if there are any issues or suggestions for improvement.

I look forward to your feedback.

Thank you!
Best regards,

File: style.css [hunk 1]
```diff
@@ -378,6 +375,19 @@ textarea {
   color: var(--content-text-color);
 }
 
+/* button hover effects */
+.contact-form .contact-btn:active {
+  background-color: var(--button-active-color);
+  color: var(--white-color);
+}
```
Comment (tresorsawasawa) [inline style.css:382]: - [ ] Good job trying to add some of changes mentioned by the previous reviewer. However, I noticed that the color you use on the `:active` state is different from the one used the original template design. I suggest you update it by using the correct one in order to match the Figma design. Here is the color to be used: `#008552`

File: style.css
Comment (RileyManda) [reply style.css:1]: > https://github.com/RileyManda/portfolio-website/blob/11bead5cfeaf890c21612829cc53956bebdb20b3/style.css#L640-L642
> 
> * [x]  Ouch! this is really close but it seems like you forgot to implement the `:hover` & `:active` states of the "Get in touch" button. So please make sure to add that
>   
>   * Here's the Figma design as a reminder:
>     firefox_kz6PEcoelc

I have completed itirations on the Portfolio’s contact section as requested by the reviewer.

Here are the itirations I made:

-Removed description/instruction text in the README file as recommended by the review.
-Added hover and active states to the contact form button in the project.

Please review the changes and let me know if there are any issues or suggestions for improvement.

I look forward to your feedback.

Thank you!
Best regards,

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @RileyManda and @Pi1998 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @RileyManda and @Pi1998 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (RileyManda) [reply style.css:382]: > * [ ]  Good job trying to add some of changes mentioned by the previous reviewer. However, I noticed that the color you use on the `:active` state is different from the one used the original template design. I suggest you update it by using the correct one in order to match the Figma design. Here is the color to be used: `#008552`


I have completed iterations on the Portfolio’s contact section as requested by the reviewer.

Here are the iterations I made:

-Updated the button active color to #008552 as per design requirements

Please review the changes and let me know if there are any issues or suggestions for improvement.

I look forward to your feedback.

Thank you!
Best regards,

**Comment by RileyManda** [comment]
I have completed iterations on the Portfolio’s contact section as requested by the reviewer.

Here are the iterations I made:

-Updated the button active color to #008552 as per design requirements

Please review the changes and let me know if there are any issues or suggestions for improvement.

I look forward to your feedback.

Thank you!
Best regards,

**Review by Zuhaib042 — APPROVED** [review]
Hi @RileyManda 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Correct use of GitHub Flow ✔
Descriptive PR title with a great summary ✔
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**